examples/complex/contracts/DonationsV1.sol

```diff
@@ -21,8 +21,7 @@ contract DonationsV1 is Ownable {
   }
 
   function withdraw(address _wallet) onlyOwner {
-
     // Withdraw all donated funds.
-    _wallet.transfer(this.balance);
+    require(_wallet.transfer(this.balance));
```

Comment:
This is not the kind of `transfer` whose return value we need to check. Only token transfers.